etl/output/test-hebis-to-lobid-output-14.json

```diff
@@ -0,0 +1,118 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/528269186#!",
+  "type" : [ "BibliographicResource", "Book" ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "title" : "Mainzer Stadtspaziergänge, Band 12: Bretzenheim, Berliner Siedlung, Hechtsheim, Laubenheim, Ebersheim / Michael Bermeitinger",
+  "almaMmsId" : "528269186",
+  "isbn" : [ "9783960310723", "3960310722" ],
+  "oclcNumber" : [ "1512396052" ],
+  "dnbId" : "1361254696",
+  "publication" : [ {
+    "startDate" : "2025",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "Roßdorf" ],
+    "publishedBy" : [ "TZ-Verlag & Print GmbH" ]
+  } ],
+  "describedBy" : {
+    "resultOf" : {
+      "object" : {
+        "id" : "http://sru.hebis.de/sru/DB=2.1?query=pica.ppn+%3D+%22528269186%22&version=1.1&operation=searchRetrieve&stylesheet=http%3A%2F%2Fsru.hebis.de%2Fsru%2F%3Fxsl%3DsearchRetrieveResponse&recordSchema=marc21&maximumRecords=10&startRecord=1&recordPacking=xml&sortKeys=LST_Y%2Cpica%2C0%2C%2C"
+      }
+    }
+  },
+  "sameAs" : [ {
+    "id" : "http://worldcat.org/oclc/1512396052",
+    "label" : "OCLC Ressource"
+  }, {
+    "id" : "https://d-nb.info/1361254696",
+    "label" : "DNB-Ressource"
+  } ],
+  "isPartOf" : [ {
+    "hasSuperordinate" : [ {
+      "id" : "http://cbsopac.rz.uni-frankfurt.de/DB=2.1/PPNSET?PPN=457093552#!",
```

Comment:
Just glossing over this: Is it intended that this id differs from the one of the resource itself which reads `"id" : "http://lobid.org/resources/457093552#!"`?

Reply:
Danke, ich verstehe das jetzt. Ist ja ganz schön vertrackt und wird vermutlich zukünftig zu weiteren Verwirrungen führen. Spontan fällt mir keine bessere Lösung ein.